src/MessagingInput/MessagingInput.tsx

```diff
@@ -141,8 +141,9 @@ const MessagingInputRenderFunction: React.ForwardRefRenderFunction<MessagingInpu
               //  explicitly set to have enter be a newline.
               if (e.key === KeyCode.ENTER && !e.shiftKey && !newlineOnEnter) {
                 e.preventDefault();
-                // If something other than whitespace is in the input area, Send the message.
-                if (value.trim() !== "") {
+                // If something other than whitespace is in the input area,
+                // or if there is at least one attachment, send the message.
+                if (value.trim() !== "" || attachments?.length) {
```

Comment:
This is a-okay, but I think `attachments?.length > 0` for the second condition might be a bit easier to read.